### PR TITLE
 Add python wrapper for 'less_than' and 'equal' compare op

### LINKED ISSUE #8299: Add python wrapper for 'less_than' and 'equal' compare op

### PR DESCRIPTION
**Pull request by wanghaoshuang**
fix #8299

### CODE REVIEW

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -945,6 +946,66 @@ def create_array(dtype):
...
+        cond = helper.create_tmp_variable(dtype='bool')
+        cond.stop_gradient = True
+
+    helper.append_op(
+        type='equal', inputs={'X': [x],
+                              'Y': [y]}, outputs={'Out': [cond]})
+    return cond
+
+
+def less_equal(x, y, cond=None, **ignored):
```
Comment (reyoung) [inline python/paddle/v2/fluid/layers/control_flow.py]: The comment does not need to write by hand.

You can import `from layer_function_generator import autodoc`, and use `autodoc()` to load C++ comments to Python.

The example is [here](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/device.py#L25)

Comment (reyoung) [reply python/paddle/v2/fluid/layers/control_flow.py]: If you want to add more comments in Python, just use

```
@autodoc(comment="""example:
   ....
""")
```

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -945,6 +946,66 @@ def create_array(dtype):
         dtype=dtype)
 
 
+def equal(x, y, cond=None, **ignored):
```
Comment (jacquesqiao) [inline python/paddle/v2/fluid/layers/control_flow.py]: `equal` is already in the control flow APIs : https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/control_flow.py#L978

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 1 above)
Comment (wanghaoshuang) [reply python/paddle/v2/fluid/layers/control_flow.py]: @reyoung  It's a cool feature. But the  `Args` in python API is different with these in C++ comments. And a formated example python code is expected. So It is more convenient  rewritting the doc in Python API.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 2 above)
Comment (wanghaoshuang) [reply python/paddle/v2/fluid/layers/control_flow.py]: @jacquesqiao Thx. I have fixed the conflicts.

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -945,6 +946,36 @@ def create_array(dtype):
         dtype=dtype)
 
 
+def less_equal(x, y, cond=None, **ignored):
```
Comment (reyoung) [inline python/paddle/v2/fluid/layers/control_flow.py]: You do not need to write documentation by hand.

Use `@autodoc()`

File: doc/api/v2/fluid/layers.rst [hunk 4]
```diff
@@ -109,6 +109,18 @@ less_than
 ..  autofunction:: paddle.v2.fluid.layers.less_than
     :noindex:
 
+equal
```
Comment (reyoung) [inline doc/api/v2/fluid/layers.rst]: THIS FILE IS GENERATED BY `gen_doc.{py|sh}`

**Pull request closed without merging**